articles/an-overview-of-fuzzy-logic-system/index.md

```diff
@@ -0,0 +1,104 @@
+Decision-making is an integral aspect of our daily lives. It is hard to establish whether a certain statement (or state) is true or false in some situations. In such situations, fuzzy logic can be used to offer flexibility in reasoning, given the uncertainty. 
+
+Fuzzy logic systems are used in automobile and domestic applications to control actions and processes. This article provides an overview of this logic and how it is implemented. It also highlights its real-life applications and explains why it is used.
+
+### What is Fuzzy Logic?
+Fuzzy means uncertain, indefinite, vague, or unclear. Fuzzy logic is a computing technique that is based on the degree of truth. A fuzzy logic system uses the input's degree of truth and [linguistic variables](https://www.igi-global.com/dictionary/granular-computing/17211) produce a certain output. The state of this input determines the nature of the output.
+
+This technique is different from [Boolean logic](https://www.bbc.co.uk/bitesize/guides/zc4bb9q/revision/1), which uses only two categories (true or false). In Boolean logic, two categories (0 and 1) are used to describe objects. For example, the temperature in water served in glass may be High (1) or Low (0). The water is described using more categories in fuzzy logic, but within the two categories mentioned earlier. In this case, the water may be very cold, very warm, or warm.
+
+Let's take another simple example. Suppose we have a question that we need to answer. In Boolean logic, the answer would be either yes or no. In fuzzy logic, the answer may be between these two categories. Some of the probable answers in this logic include possibly yes, possibly no, or certainly no. 

```

Comment:
*boolean logic*. Change such occurrences throughout the article.

Reply:
Done.